index.mako

```diff
@@ -1633,6 +1637,28 @@ immutable caches:
         key: pre-commit|${{ env.PY }}|${{ hashFiles('.pre-commit-config.yaml') }}
 ```
 
+### circleci example
+
+like [azure pipelines](#azure-pipelines-example), circleci also uses
+immutable caches:
+
+```yaml
+steps:
+    - run:
+        command: |
+            cat .pre-commit-config.yaml > pre-commit-deps.txt
```

Comment:
UUOC: `cp` does this better

but why make a separate file, can't you just use the original file?

also looks like `pre-commit-deps.txt` is unused

Reply:
This was a copy-pasta error. Creating a new file, `pre-commit-deps.txt`, to concatenate `.pre-commit-config.yaml` & `python -VV`, since CircleCI can't use environment variables in cache key templates.